server/service/src/processors/transfer/invoice/update_inbound_invoice.rs

```diff
@@ -93,10 +93,41 @@ impl InvoiceTransferProcessor for UpdateInboundInvoiceProcessor {
             invoice_line_repository.upsert_one(line)?;
         }
 
+        let outbound_invoice_row = &outbound_invoice.invoice_row;
+
+        let formatted_ref = match &outbound_invoice_row.their_reference {
+            Some(reference) => format!(
+                "From invoice number: {} ({})",
```

Comment:
the returning nagging thought that we need to have translatable text in the backend, somehow